Issue 970, Adding Ideal values while testing Peerconnection Audio

rantonysamy opened the pull request:
Description
Issue 970, Adding Ideal values while testing Peerconnection Audio / Different CodeX

Purpose
Added Values for the Below Codex
Opus | iSAC 16K | G722 | PCMU

Commit:
Issue 970, Adding Ideal values while testing Peerconnection Audio / Different CodeX

rantonysamy commented:
I will be sending a fixup for this tomorrow.

Commit:
Fixup: Issue 970, Fixing the test a bit

Commit:
Fixup2: Removing an Extra Char, so that the cont.Integ. could pass

KaptenJansson commented on src/content/peerconnection/audio/index.html:
```diff
@@ -32,50 +32,66 @@
 
 <body>
 
-  <div id="container">
```
Please indent this again, it belongs to \<body\>. The HTML linter is not strict enough or is not running for these files it seems.

KaptenJansson commented on src/content/peerconnection/audio/index.html:
```diff
@@ -32,50 +32,66 @@
+</div>
+<hr>
+<table>
+  <caption>Bitrate and Packes sent per second - approximate results in Chrome, Opera, Firefox, Safari</caption>
```
If the numbers below have been measured manually in the browsers mentioned, please include the version numbers of the browsers in case things change.

rantonysamy commented:
Okey will do these changes.

Commit:
Fixup3: Added Indentation and tested browser names

KaptenJansson commented on src/content/peerconnection/audio/index.html:
```diff
@@ -78,12 +73,14 @@ <h1><a href="//webrtc.github.io/samples/" title="WebRTC samples homepage">WebRTC
+    <td>~30 kbps / Muted : Same, ~33 Packets, Muted : Same or slight drop</td>
     <td>~70 kbps / Muted : Same, ~50 Packets, Muted : Same</td>
     <td>~70 kbps / Muted : Same, ~55 Packets, Muted : Same</td>
+    <td>Tested in Chrome, Not tested in Opera, Firefox, Safari, Edge</td>
```
What version of Chrome?

fippo commented on src/content/peerconnection/audio/index.html:
```diff
@@ -27,55 +26,69 @@
-      </select>
-      <button id="callButton">Call</button>
-      <button id="hangupButton">Hang Up</button>
+<div id="container">
```
looks like you changed indent here?

KaptenJansson replied:
Yes please check that the indentation is correct, otherwise it's a mess to read.

rantonysamy replied:
Sure will test it once and let you know everything okey.

rantonysamy replied:
Everything is fine in-fact, no issues found. kindly check you can approve.

fippo replied:
but indent changes from 2 spaces to 0 spaces (for this and everything else) -- which makes the diff much harder to read
(and yes, indent has been inconsistent before)

KaptenJansson replied:
Please indent correctly, it should belong to the body tag.

Commit:
Fixup4: Checked Indents and removed one HR / some text, looks good

KaptenJansson commented on src/content/peerconnection/audio/css/main.css:
```diff
@@ -17,6 +17,16 @@ button {
 }
 
+table {
+   border-collapse: collapse;
```
Should have 2 indents like the rest.

KaptenJansson commented on src/content/peerconnection/audio/css/main.css:
```diff
@@ -17,6 +17,16 @@ button {
+  border: 1px solid black;
+}
+
+tr:hover {background-color:#f5f5f5;}
```
Put the property on a separate line like the rest.

rantonysamy replied:
Done.

Commit:
Fixup4: Indentation and other reviewes fixed

Commit:
FixupTry5 : Issue with Indents

KaptenJansson commented on src/content/peerconnection/audio/index.html:
```diff
@@ -53,38 +53,37 @@ <h1><a href="//webrtc.github.io/samples/" title="WebRTC samples homepage">WebRTC
+    </tr>
+  </table>
+  <hr>
+  <script src="https://webrtc.github.io/adapter/adapter-latest.js"></script><script src="../../../js/common.js"></script>
```
<script src="../../../js/common.js"></script> should be on a new line.

Commit:
Fixup6 : Added new line as requested

Commit:
Fixup7 : Added -in-a-new line as requested

Commit:
Fixup8 : removed two script tags in 1 line line as requested

KaptenJansson commented on src/content/peerconnection/audio/css/main.css:
```diff
@@ -17,6 +17,18 @@ button {
+}
+
+tr:hover {
+  background-color:#f5f5f5;
```
Missing space after `:`.

KaptenJansson commented on src/content/peerconnection/audio/index.html:
```diff
@@ -27,55 +26,67 @@
-      <div>
-        <div class="label">Remote audio:</div><audio id="audio2" autoplay controls></audio>
-      </div>
+  <h1><a href="//webrtc.github.io/samples/" title="WebRTC samples homepage">WebRTC samples</a> <span>Peer connection: audio only</span></h1>
```
Indent is incorrect, should belong to the container div. Applies throughout the file.

rantonysamy committed:
Update main.css

rantonysamy committed:
Fixup: Fixing the table alignment and Div

All the Alignment is just same as the previous file, only the table has been added.

rantonysamy committed:
Fixup: Updating the Fixup with removing space

Updating the Fixup by removing one extra linefeed.

rantonysamy committed:
Fixing the colon in background color (hover).

Fixup for the line 28, removed the wrong spacing and adding the space in the correct place (bg)

KaptenJansson reviewed (approved):
Test errors unrelated to this. LGTM
